File: Lancetilla BackEnd/Lancetilla/Lancetilla.API/Models/CuidadosViewModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace Lancetilla.API.Models
{
    public class CuidadosViewModel
    {

        public int cuid_Id { get; set; }
        public string cuid_Observacion { get; set; }
        public int ticu_Id { get; set; }
        public string ticu_Descripcion { get; set; }
        public int? cuid_UserCreacion { get; set; }
        public DateTime? cuid_FechaCreacion { get; set; }
        public int? cuid_UserModificacion { get; set; }
        public DateTime? cuid_FechaModificacion { get; set; }

    }
}

File: Lancetilla FrontEnd/src/app/demo/Models/CuidadosViewModel.ts
export class CuidadosViewModel {
    constructor(
        public cuid_Id?: number,
        public cuid_Observacion?: string,
        public ticu_Id?: number,
        public ticu_Descripcion?: string,
        public usua_UserCreaNombre?: string,
        public cuid_UserCreacion?: number,
        public cuid_FechaCreacion?: string,
        public usua_UserModiNombre?: number,
        public cuid_UserModificacion?: number,
        public cuid_FechaModificacion?: string,
        public cuid_Estado?: boolean
    ){}
}